### ISSUE TITLE
bug: 'Topic does not exist' in test_file_put_large

### ISSUE THREAD

**Issue by hannes-ucsc** [HumanCellAtlas/data-store#628]
2637b94, master

```
# make test
<snip>
coverage run -p --source=dss -m unittest tests/test_file.py
......Response:
{'code': 'unhandled_exception',
 'stacktrace': 'Traceback (most recent call last):\n'
               '  File "/Users/hannes/workspace/hca/data-store/dss/error.py", '
               'line 26, in wrapper\n'
               '    return func(*args, **kwargs)\n'
               '  File '
               '"/Users/hannes/workspace/hca/data-store/dss/api/files.py", '
               'line 191, in put\n'
               '    task_id = '
               'chainedawslambda.aws.schedule_task(s3copyclient.S3CopyWriteBundleTask, '
               'state)\n'
               '  File '
               '"/Users/hannes/workspace/hca/data-store/.venv/lib/python3.6/site-packages/chainedawslambda/aws.py", '
               'line 69, in schedule_task\n'
               '    Message=json.dumps(payload),\n'
               '  File '
               '"/Users/hannes/workspace/hca/data-store/.venv/lib/python3.6/site-packages/botocore/client.py", '
               'line 312, in _api_call\n'
               '    return self._make_api_call(operation_name, kwargs)\n'
               '  File '
               '"/Users/hannes/workspace/hca/data-store/.venv/lib/python3.6/site-packages/botocore/client.py", '
               'line 601, in _make_api_call\n'
               '    raise error_class(parsed_response, operation_name)\n'
               'botocore.errorfactory.NotFoundException: An error occurred '
               '(NotFound) when calling the Publish operation: Topic does not '
               'exist\n',
 'status': 500,
 'title': 'An error occurred (NotFound) when calling the Publish operation: '
          'Topic does not exist'}
F...
======================================================================
FAIL: test_file_put_large (tests.test_file.TestFileApi)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/hannes/workspace/hca/data-store/tests/test_file.py", line 77, in test_file_put_large
    self._test_file_put_large("aws", "s3", self.s3_test_bucket, S3Uploader(tempdir, self.s3_test_bucket))
  File "/Users/hannes/workspace/hca/data-store/tests/test_file.py", line 92, in _test_file_put_large
    resp_obj = self.upload_file_wait(f"{scheme}://{test_bucket}/{src_key}", replica, expect_async=expect_async)
  File "/Users/hannes/workspace/hca/data-store/tests/infra/upload_mixin.py", line 38, in upload_file_wait
    source_url=source_url,
  File "/Users/hannes/workspace/hca/data-store/tests/infra/assert_mixin.py", line 75, in assertResponse
    self.assertEqual(response.status_code, expected_code)
AssertionError: 500 != 202

----------------------------------------------------------------------
Ran 10 tests in 53.218s

FAILED (failures=1)
make: *** [tests/test_file.py] Error 1
```

**Comment by ttung** [comment]
This test requires the lambdas be deployed and operational.

**Comment by hannes-ucsc** [comment]
The README instructs to invoke `make test` first and only proceed to `make deploy` if the tests pass. Traditionally, if a test depends on a finished deployment, it is not a unit test. We should make the test conditional or at least mention this caveat in the README.  I'd be happy to implement either solution.

**Comment by hannes-ucsc** [comment]
Sorry if this seems pedantic. If there are more tests like this that depend on a deployment, maybe the right solution is to change the README to run `make deploy` first.

**Comment by hannes-ucsc** [comment]
Fixed by #779
